lib/veksel/cli.rb

```diff
@@ -14,8 +14,12 @@ def self.fork
       require 'psych'
       require 'ostruct'
       require 'fileutils'
+      require 'erb'
+
+      config = Psych.safe_load(
+        ERB.new(File.read('config/database.yml')).result, aliases: true, symbolize_names: true
+      )[:development]
```

Comment:
```suggestion
      raw_config = File.read('config/database.yml')
      if raw_config.scan('<%=').any?
        require 'erb'
        raw_config = ERB.new(raw_config).result
      end

      config = Psych.safe_load(
        raw_config, aliases: true, symbolize_names: true
      )[:development]
```

Feel free to use something else to detect erb usage.